ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -418,7 +418,7 @@ extension SignalProtocol where Value: OptionalProtocol, Value.Wrapped: AnyObject
 }
 
 extension Action {
-	fileprivate var isEnabled: RACSignal<NSNumber> {
+	fileprivate var _isEnabled: RACSignal<NSNumber> {
```

Comment:
Nit: rename to `isEnabledSignal`?

Reply:
Or possibly `racIsEnabled` or `rac_isEnabled`? 🤔

Reply:
Yeah, I'm happy with `isEnabledSignal` 👍  thanks!